src/styles/blocks/card.scss

```diff
@@ -0,0 +1,77 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+
+  &_name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+    margin-bottom: 4px;
+  }
+
+  &_code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &_image {
+    width: 160px;
+    height: 134px;
+    margin: 16px auto 40px;
+  }
+
+  &_price-container {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &_buy-button {
+    display: flex;
+    background-color: #00acdc;
+    color: white;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    border: 1px solid transparent;
+    border-radius: 5px;
+    width: 166px;
+    height: 40px;
+    justify-content: center;
+    align-items: center;
+    margin-top: 16px;
+  }
+
+  &_buy-button:hover {
+    background-color: white;
+    color: #00acdc;
+    border-color: #00acdc;
+  }
+
+  &--stars {
+    margin-bottom: 24px;
+    margin-top: 16px;
+  }
+
+  &_price {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+  }
+
+  &_number {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
```

Comment:
Again BEM, you added a margin to `.catalog__card` and u should remove it for `.catalog__card` but not for the card. Check the comment for the file catalog.scss

Reply:
ok, thanks, I wronly thought that this is internal style for card